_includes/runs-new.md

```diff
@@ -2,37 +2,43 @@
 
 # 3. Creating runs from scratch
 
-MISO supports runs from both Illumina and PacBio sequencers (and others), so 
+MISO supports runs from Illumina, PacBio, Oxford Nanopore, and other sequencers, so 
 the terms used for instrument runs and associated libraries are intentionally 
-different from those used by the vendor.
-Every time a sequencer is loaded and sequencing begins,
-a _Run_ is created. Runs are picked up automatically from the instrument.
-A _Sequencing Container_ is the link between the library information and the
-instrument Run and contains one or more lanes. Each Illumina lane, or
-PacBio SMRT cell in the container is loaded with exactly one _Pool_.  Runs and
-Containers can be associated as soon as the Run and Container are both created.
+different from those used by the vendor. Every time a sequencer is loaded and
+sequencing begins, a _Run_ is created. Illumina and PacBio Runs are picked up
```

Comment:
Wording thoughts unrelated to the things you've changed:
"Every time a sequencer is loaded and sequencing begins, a _Run_ is created" -> "A MISO _Run_ represents a sequencer that has been loaded and sequencing has begun" ?
"Illumina and PacBio Runs are picked up automatically from the instrument, [...]" -> "The run scanner can be configured to detect new Illumina and PacBio runs and create corresponding Runs in MISO, [...]"